lib/panel-utils.js

```diff
@@ -324,6 +317,32 @@ function messageHandler(opts) {
   }
 }
 
+// Opens the video in the original tab, if it is still open, else in a new tab.
+function sendToTab(panel, opts) {
+  const pageUrl = getPageUrl(opts.domain, opts.id, opts.time);
+
+  if (pageUrl) {
+    const tabs = require('sdk/tabs');
+    let targetTab = null;
+    for (let tab of tabs) { // eslint-disable-line prefer-const
+      if (tab.id === opts.tabId) {
+        targetTab = tab;
+        break;
+      }
+    }
+    if (targetTab !== null) {
+      targetTab.url = pageUrl;
```

Comment:
this should be `targetTab.activate()`
the above code only refreshes the tab, it doesn't make it active.